packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts

```diff
@@ -479,6 +480,758 @@ export = {
 
       test.done();
     },
+    'with VPC configuration'(test: Test) {
+      const stack = new cdk.Stack();
+
+      const bucket = new s3.Bucket(stack, 'MyBucket');
+      const vpc = new ec2.VpcNetwork(stack, 'MyVPC');
+      const securityGroup = new ec2.SecurityGroup(stack, 'SecurityGroup1', {
+          groupName: 'Bob',
+          vpc: vpc,
+          allowAllOutbound: true,
+          description: 'Example',
+      });
+      new codebuild.Project(stack, 'MyProject', {
+        source: new codebuild.S3BucketSource({
+          bucket,
+          path: 'path/to/source.zip',
+        }),
+        vpc: vpc,
+        securityGroups:[securityGroup]
+      });
+
+      expect(stack).toMatch({
```

Comment:
I think this is a bit much for a unit test 😲 How about using the `expect(stack).to(haveResourceLike(...` functionality in our helpers, and making this assertion smaller (limited to only the Project props)?